### ISSUE TITLE
Contact info

### ISSUE THREAD

**Issue by arneke** [PDOK/mapserver-docker#15]
Thanks for sharing. The email in the Dockerfile doesn't work, where can I contact someone regarding a very minor security issue?

**Comment by WouterVisscher** [comment]
Hi @arneke, through this issue I presume 😄
regarding your "second" issue, the contact informatie, I will look into it if we can 'revive' the email adres.
Most effective 'proces' will be through the github issues

**Comment by arneke** [comment]
Well, if you are okay with public, it is easy to fix either way:

The MAP filter does not really work, you can just append multiple MAP parameters. Based on my testing, the last one is actually used. So a quick fix would be to append the correct one to the end. May not be robust though.

**Comment by WouterVisscher** [comment]
oke, we will look into that
thanks for the information

**Comment by WouterVisscher** [comment]
@arneke we moved from the regex in the lighttpd to a lua script solution. All 'MAP=' query key(and value) are now removed. Getting the email adres back wasn't a viable solution so we updated the README.md

If you would like to test this against https://github.com/PDOK/mapserver-docker/tree/7.6.3 we would appreciate it. Maybe we overlook something, but as far as we see it's not possible to parse additional MAP= (or any for that matter) parameters in the KVP Query.
